server/bin/gold/test-5.2.txt

```diff
@@ -1,4 +1,12 @@
 +++ Running pbench-server-prep-shim-002
+Warning no default label for /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-0_1970.01.01T00.42.00.tar.xz
+Warning no default label for /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-0_1970.01.01T00.42.00.tar.xz.md5
+Warning no default label for /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-w-dot-prefix_1970.01.01T00.42.00.tar.xz
+Warning no default label for /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-w-dot-prefix_1970.01.01T00.42.00.tar.xz.md5
+Warning no default label for /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-w-prefix-dot_1970.01.01T00.42.00.tar.xz
+Warning no default label for /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-w-prefix-dot_1970.01.01T00.42.00.tar.xz.md5
+Warning no default label for /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-g-normal/tarball-normal_1970.01.01T00.42.00.tar.xz
+Warning no default label for /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-g-normal/tarball-normal_1970.01.01T00.42.00.tar.xz.md5
```

Comment:
Who generates this output?  If this is coming from the selinux module itself, we'll have to get a hold of how it is produced.

Reply:
yeah It is coming from the SELinux module I will see how it is getting produced.

Reply:
"libselinux: selinux_restorecon: only log no default label warning for caller-supplied pathname", I will look how we can disable this warning

Reply:
I believe this has been resolved.